src/util/prefer-v2-deployment.ts

```diff
@@ -16,7 +16,7 @@ export async function hasServerfile(cwd: string) {
   return new Promise(res => exists(join(cwd, 'server.js'), res));
 }
 
-const PREFIX = `Changing platform version to ${code('2')}`;
+const INFO = `More details: ${link('https://zeit.co/docs/version-detection')}`;
```

Comment:
This URL is a 404 currently.

Reply:
@TooTallNate yup, it's in the process of getting added: https://github.com/zeit/docs/pull/1089